### PR TITLE
[MAG-143] Using Plugin to Refresh Tax when `Store Front` Currency Sel…

### PR DESCRIPTION
**Pull request by suraiyavic**
…ection Changes

### CODE REVIEW

File: Model/DutyCalculator.php [hunk 1]
```diff
@@ -213,7 +213,7 @@ public function handleCaseWithCountryAndStateSpecified($address, $apply)
         $quote = $this->checkoutSession->getQuote();
         $this->response->setDuty($quote->getReachDuty());
         $this->handleTaxApplicability($address, $apply);
-        $this->_logger->debug('Country is specified but state is not selected (when both are needed); so we would'.
+        $this->_logger->debug('Country is specified but state is not selected (when both are needed) or the selection of (country, state) combo didnot change; so we would'.
```
Comment (lizzyWithReach) [inline Model/DutyCalculator.php]: tiny tiny, "didnot" -> "did not"

**Review by lizzyWithReach — APPROVED** [review]
tiny comment typo, approving the review so that if you choose to fix it you can just merge that change here and get it into master. Thank you!

File: Model/DutyCalculator.php (hunk 1 above)
Comment (suraiyavic) [reply Model/DutyCalculator.php]: Commit 148d4bd172dd1829887869ac126b5f2e7f67bc95
Thanks.

**Review by lizzyWithReach — APPROVED** [review]
Thanks!

**Pull request merged**